plans/smlbench2/main.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"time"
+
+	utils "github.com/ipfs/testground/plans/smlbench2/utils"
+	iptb "github.com/ipfs/testground/sdk/iptb"
+	"github.com/ipfs/testground/sdk/runtime"
+	"github.com/ipfs/testground/sdk/sync"
+	"github.com/libp2p/go-libp2p-core/peer"
+	// shell "github.com/ipfs/go-ipfs-api"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/ipfs/go-cid"
+)
+
+const (
+	// sizeBytes = 10 * 1024 * 1024 // 10mb
+	sizeBytes = 100 * 1024 * 1024 // 100mb
+)
+
+var cidSubtree = &sync.Subtree{
+        GroupKey:    "cid",
+        PayloadType: reflect.TypeOf(&cid.Cid{}),
+        KeyFunc: func(val interface{}) string {
+                return "cid"
+        },
+}
+
+func main() {
+	runenv := runtime.CurrentRunEnv()
+	if runenv.TestCaseSeq < 0 {
+		panic("test case sequence number not set")
+	}
+
+	timeout := func() time.Duration {
+                if t, ok := runenv.IntParam("timeout_secs"); !ok {
+                        return 30 * time.Second
+                } else {
+                        return time.Duration(t) * time.Second
+                }
+        }()
+
+	watcher, writer := sync.MustWatcherWriter(runenv)
+        defer watcher.Close()
+
+	spec := iptb.NewTestEnsembleSpec()
+	spec.AddNodesDefaultConfig(iptb.NodeOpts{Initialize: true, Start: true}, "local")
```

Comment:
Should this be "AddNodes" or "AddNode" (singular)? Given that it will only run a node per container?